siliconcompiler/remote/client.py

```diff
@@ -169,10 +170,11 @@ def _remote_preprocess(chip, steplist):
     # we need to send inputs up to the server.
     chip._collect()
 
-    # Set 'steplist' to only the remote steps, for the future server-side run.
+    # Set 'from/to' to only the remote steps, for the future server-side run.
     chip.unset('arg', 'step')
     chip.unset('arg', 'index')
-    chip.set('option', 'steplist', remote_steplist, clobber=True)
+    chip.set('option', 'from', [remote_nodelist[0][0]], clobber=True)
+    chip.set('option', 'to', [remote_nodelist[-1][0]], clobber=True)
```

Comment:
I don't think you can assume any ordering here.
You probably need to use exit nodes as the 'to' and the outputs from the entry nodes as a 'from'

Reply:
Changed it so it starts off at any successor of an entry node.